solutions/avnm-secured-hub-and-spoke/bicep/main.bicep

```diff
@@ -8,11 +8,10 @@ param resourceGroupName string
 
 @description('The location of this regional hub. All resources, including spoke resources, will be deployed to this region.')
 @minLength(6)
-param location string
+param location string = deployment().location
```

Comment:
can we please test deploying to westus instead?

Reply:
The main bicep is at subscription level because of policy creation. I decided to create the resource group inside the bicep file to avoid conflicts detected (good cacth). The location is taken from the deployment location for the resource group and all the resources are in the resource group location. The name of the resource group contains the suffix -location.

I changed the EncriptionAtHost to false as requested. 

The readme was updated accordingly